src/toolchain/rust.rs

```diff
@@ -409,6 +409,22 @@ fn install_rust_nightly(version: &str) -> Result<()> {
     Ok(())
 }
 
+/// Unnstalls the RiscV target.
+pub fn uninstall_riscv_target(nightly_version: &str) -> Result<()> {
+    info!("{} Installing Riscv target", emoji::WRENCH);
```

Comment:
```suggestion
    info!("{} Uninstalling Riscv target", emoji::WRENCH);
```

Typo here, also we seem to be printing "Deleteing RISC-V target" and then immediately printing "Installing RiscV target".

Additionally, I think we should be consistent with how we display these; __RISC-V__ is the "official" way to write this so maybe we should stick with that.